homeapp/templates/base.html

```diff
@@ -162,16 +162,16 @@ <h4 class="modal-title">Login with</h4>
                         <div class="flex-caption">
                             {% block header-text %}
                             {% endblock %}
-                            <form method="GET" action="{% url 'searchapp:search' %}"
+                            <form method="GET" action="{% url 'search:search' %}"
                                   class="form-inline row input-group" id="search">
+                                {% csrf_token %}
```

Comment:
It is not necessary. You can put it anywhere between form tag. It is a standard way to write just after opening form tag (<form>)